ksql-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -165,12 +165,25 @@
       "Config to enable or disable transient pull queries on a specific KSQL server.";
   public static final boolean KSQL_QUERY_PULL_ENABLE_DEFAULT = true;
 
+  // Shall we remove this config?
   public static final String KSQL_QUERY_PULL_ROUTING_TIMEOUT_MS_CONFIG =
       "ksql.query.pull.routing.timeout.ms";
   public static final Long KSQL_QUERY_PULL_ROUTING_TIMEOUT_MS_DEFAULT = 30000L;
   public static final String KSQL_QUERY_PULL_ROUTING_TIMEOUT_MS_DOC = "Timeout in milliseconds "
       + "when waiting for the lookup of the owner of a row key";
 
+
+  public static final String KSQL_QUERY_PULL_ALLOW_STALE_READS =
```

Comment:
To be consistent with other names, maybe something like "ksql.query.pull.stale.reads.enabled"

Reply:
Done